_opcodes/op201.html

```diff
@@ -11,3 +11,9 @@
 pst: 0
 ---
 The targeted creature(s) will absorb a number of spells (specified by the 'Amount' field), from the spell level specified in the 'Level' field. A spell is assigned to a spell level by the power level of the first effect greater than the 'Level' parameter in the extended header that is being called.
+
+{% capture note %}
+It cannot deflect effects through AoE projectiles.
```

Comment:
```suggestion
It cannot absorb AoE spells.
```
Even then I'm not sure — it likely does if the spell is directly targetted.

Reply:
> Even then I'm not sure — it likely does if the spell is directly targetted.

Do you mean it likely does if the SPL is directly targeted to a specific creature?

Reply:
That's my guess. Casting a fireball on someone with spell trap, does it eat it up?